inventory/sample.gcp.example.com.d/inventory/gce.ini

```diff
@@ -41,9 +41,9 @@ libcloud_secrets =
 
 # If you are not going to use a 'secrets.py' file, you can set the necessary
 # authorization parameters here.
-gce_service_account_email_address = <REPLACE WITH A VALID SERVICE ACCOUNT E-MAIL ADDRESS>
-gce_service_account_pem_file_path = <REPLACE WITH A VALID SERVICE ACCOUNT PEM/JSON FILE>
-gce_project_id = <REPLACE WITH A VALID PROJECT ID>
+gce_service_account_email_address = "{{ lookup('env','GCE_EMAIL') }}" 
```

Comment:
Curious about using environment variables for this. That means that they could change for a follow-up run. Is that going to be a problem? May want to just keep them as part of the inventory if so.